scripts/build_script.sh

```diff
@@ -37,6 +38,9 @@ while getopts "cdhps:" option; do
         s) # Debian package version suffix
             PACKAGE_VERSION_SUFFIX="-DPACKAGE_VERSION_SUFFIX=${OPTARG}"
             ;;
+        D) # Pass CMake options
+            cmake_options+=("-D$OPTARG")
```

Comment:
maybe `cmake_options` variable can be all upper case here to be consistent?

Reply:
Sure I can change this